<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
		<!-- node parameters -->
		<param name="serial_name" type="string" value="/dev/ttyACM0"/>
		<param name="baud_rate" type="int" value="115200"/>
		<param name="app_id" type="int" value="1022384"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="Welcome to use dji-sdk"/>
		<param name="enc_key" type="string" value="e7bad64696529559318bb35d0a8c6050d3b88e791e1808cfe8f7802150ee6f0d"/>
		<param name="groundstation_enable" type="int" value="1"/>
		<param name="uart_or_usb" type="int" value="1"/>
		<param name="A3_or_M100" type="int" value="0"/>
	</node>
</launch>
